src/compiler.md

```diff
@@ -96,8 +96,8 @@ There are several flags that enable or disable specific WebAssembly or compiler
                         stub         Minimal runtime stub (never frees)
                         ...          Path to a custom runtime implementation
 
---exportRuntime       Always exports the runtime helpers (__new, __collect etc.).
-                      When host `bindings` is enabled it will be determined automatically.
+--exportRuntime       Always exports the runtime helpers (__new, __collect, __pin etc.).
+                      Automatically determined when generation of --bindings is enabled.
```

Comment:
Only noticing now: I think the lines here have become (too) long?

Reply:
No, I guess it's ok